bin/build-scripts/ftl-to-js.js

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env node
+
+// This is called by make. argv[2] is the destination directory. The the args
+// after are the paths to ftl files.
+//
+// The destination directory has been created at this point.
+
+const fs = require('fs');
+const destDir = process.argv[2];
+const ftlPaths = process.argv.slice(3);
+
+for (let ftl of ftlPaths) {
+  let locale = ftl.split('/')[1];
+  let destFilePath = `${destDir}/${locale}.js`;
+  let ftlFileContent = fs.readFileSync(ftl, 'utf8');
+  let jsFileContent = `
+    (context => {
```

Comment:
I'm not 100% sure where this file is built into, and then whether it gets translated with babel – but if it doesn't go through babel then the ES6 syntax here will cause problems on some browsers.